File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 6/11/2026, 11:34:17 AM
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        missing_no = 0
        for i in range(len(nums)+1):
            missing_no ^= i

        for i in nums:
            missing_no ^= i
        return missing_no

File: 268-MissingNumber/268-MissingNumber.cpp
// Last updated: 6/11/2026, 11:34:11 AM
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int xorr = 0;
        for(int i=0; i<nums.size(); i++){
            xorr = xorr ^ (nums[i]^i);  
        }

        return nums.size()^xorr;
    }
};